webcompat/helpers.py

```diff
@@ -100,6 +100,18 @@ def get_user_info():
         session['avatar_url'] = gh_user.get('avatar_url')
 
 
+def version_string(json_section):
+    '''Gives version string out of section with `major`, `minor`, and `patch`.'''
```

Comment:
(section here is confusing as well, so this should be updated to reflect what we rename the argument to)

The other `get_foo` methods in this file generally start like `Return blah from`. Can we follow that convention as well?

Also, the comment should ideally indicate that it will only return minor and patch if they exist.